Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -229,4 +279,51 @@ lemma free_bot (h : A ≠ ⊥) : A.Free (⊥ : SimpleGraph β) := by
 
 end Free
 
+/-!
+#### Induced containment
+
+A graph `H` *inducingly contains* a graph `G` if there is some graph embedding `G ↪ H`. This amounts
+to `H` having an induced subgraph isomorphic to `G`.
+
+We denote "`G` is contained in `H`" by `G ⊴ H` (`\triangle_left_eq`).
+-/
+
+/-- A simple graph `G` is contained in a simple graph `H` if there exists an induced subgraph of `H`
+isomorphic to `G`. This is denoted by `G ⊴ H`. -/
+def IsIndContained (G : SimpleGraph V) (H : SimpleGraph β) : Prop := Nonempty (G ↪g H)
```

Comment:
The type variable choices here are weird...

Reply:
βhat do you mean?